monitor.go

```diff
@@ -163,24 +171,30 @@ func (p *stageMonitor) draw(dc drawContext) {
 	default:
 		x, y := int(p.x), int(p.y)
 		labelRender := gdi.NewTextRender(defaultFont2, 0x80000, 0)
+		labelRender.Scale = p.size
 		labelRender.AddText(p.label)
 		labelW, h := labelRender.Size()
 
 		textRender := gdi.NewTextRender(defaultFontSm, 0x80000, 0)
```

Comment:
And is it possible to specify font file? Or we should wait for

> the upgrade of the font library to EbitenEngine or Godot.

and then support it?